docs/api.md

```diff
@@ -250,12 +250,26 @@ See [Custom Tags](#custom-tags).
 {% endapi %}
 
 {% api %}
+removeExtension
+env.removeExtension(name)
+
```

Comment:
For `removeExtension`, I copied/paste the doc for `addExtension`, which contain blank line. For `hasExtension`, I copied the one from `getExtension`, which doesn't contain blank lines.